index.html

```diff
@@ -652,19 +652,33 @@ <h3 class="tooling__title">
      <p class="tooling__text">Flexible and scalable tool to provide RSpec integration tests using definition files such as API Blueprint</p>
     </div></li>
 
-        <li class="tooling__item"><div class="tooling__item--wrap">
-            <span class="tooling__ico">
-                <span class="tooling__ico__logo tooling__ico--construction tooling__paw"></span>
-            </span>
-            <h3 class="tooling__title">
-                <a href="http://blog.apiary.io/2014/12/02/Paw-Integration/" class="block larger">
-                    <span class="block">Paw</span>
-                    <span class="tooling__link"></span>
-                </a>
-                <span class="block smaller">"The ultimate REST client for Mac"</span>
-            </h3>
-            <p class="tooling__text">A fully-featured HTTP client for the Mac featuring API Blueprint integration via extensions.</code></p>
-        </div></li>
+	<li class="tooling__item"><div class="tooling__item--wrap">
```

Comment:
Can you please use spaces instead of hard tabs, to stay consistent with the rest of the file? Thanks!